### ISSUE TITLE
Left-typed variable declaration (Dim)

### ISSUE THREAD

**Issue by Kr00l** [twinbasic/lang-design#16]
**Is your feature request related to a problem? Please describe.**
This idea comes from twinbasic/twinbasic#79 where Olaf raised a "nice to have" feature. (https://github.com/WaynePhillipsEA/twinbasic/issues/79#issuecomment-825377802)

**Describe the solution you'd like**
To specify the "As [Type]" only once on the left side and have then all the assigned variables on the right side.
`Dim As Long i, j, k`

**Additional context**
This seems to be a feature from FreeBasic.
Additional question whether or not to allow identifier type character on this left-typed declaration.
E.g. `Dim & i, j`

**Comment by Greedquest** [comment]
How will this play with existing Dim statement?

```vb
Dim As Long i, j, k, v As Variant
Dim As Long i, j, k, As Double x, y
Dim As Long i, j, k As Double, x, y
```

**Comment by EduardoVB** [comment]
An error? 
Once you defined the type at the start, then all the line is of that type.
It avoids misunderstandings. If you want, you can place a : and add another Dim with another type.
(my opinion)
